purescript.cabal

```diff
@@ -119,66 +119,66 @@ common defaults
     -- specific version.
     aeson >=1.5.6.0 && <1.6,
     aeson-better-errors >=0.9.1.0 && <0.10,
-    aeson-pretty >=0.8.8 && <0.9,
-    ansi-terminal >=0.10.3 && <0.11,
+    aeson-pretty >=0.8.9 && <0.9,
+    ansi-terminal ==0.11.*,
     array >=0.5.4.0 && <0.6,
-    base >=4.14.1.0 && <4.15,
+    base >=4.14.3.0 && <4.15,
     base-compat >=0.11.2 && <0.12,
     blaze-html >=0.9.1.2 && <0.10,
     bower-json >=1.0.0.1 && <1.1,
     boxes >=0.1.5 && <0.2,
     bytestring >=0.10.12.0 && <0.11,
     Cabal >=3.2.1.0 && <3.3,
-    cborg >=0.2.4.0 && <0.3,
-    serialise >=0.2.3.0 && <0.3,
+    cborg >=0.2.6.0 && <0.3,
+    serialise >=0.2.4.0 && <0.3,
     cheapskate >=0.1.1.2 && <0.2,
-    clock ==0.8.*,
-    containers >=0.6.2.1 && <0.7,
-    cryptonite ==0.27.*,
+    clock >=0.8.2 && <0.9,
+    containers >=0.6.5.1 && <0.7,
+    cryptonite ==0.29.*,
     data-ordlist >=0.4.7.0 && <0.5,
     deepseq >=1.4.4.0 && <1.5,
     directory >=1.3.6.0 && <1.4,
-    dlist >=0.8.0.8 && <0.9,
+    dlist ==1.0.*,
     edit-distance >=0.2.2.1 && <0.3,
-    file-embed >=0.0.13.0 && <0.1,
+    file-embed >=0.0.15.0 && <0.1,
     filepath >=1.4.2.1 && <1.5,
     fsnotify >=0.3.0.1 && <0.4,
     Glob >=0.10.1 && <0.11,
-    haskeline >=0.8.1.1 && <0.9,
-    language-javascript ==0.7.0.0,
-    lifted-async >=0.10.1.3 && <0.11,
+    haskeline >=0.8.2 && <0.9,
+    language-javascript ==0.7.1.0,
+    lifted-async >=0.10.2.2 && <0.11,
```

Comment:
We started pinning to an exact version of `language-javascript` because of https://github.com/purescript-web/purescript-web-html/issues/31; we didn't want some people's PS compilers supporting some foreign code that other people's PS compilers didn't support. I don't think there'd be a problem with bumping to the next minor version provided we trust the semantic versioning of `language-javascript` and exercise due diligence; that package is basically part of the compiler's interface to users.

Reply:
Ah, thanks.
[Here's all the changes made between `0.7.0.0` and `0.7.1.0`](https://github.com/erikd/language-javascript/compare/de4db524e7987436d618b0b53a38266727414955..376b83afdba022da1462f9814e63f2c5a2b15120). As far as I can tell, this should be safe to do.

Reply:
Here's the PR that adds support for `async` and `await`: https://github.com/erikd/language-javascript/pull/113

It looks like a new constructor is added to an existing data type, which I thought required a major version bump according to the PVP.  I decided to ask on the Haskell discourse about this: https://discourse.haskell.org/t/does-adding-a-data-constructor-require-a-major-version-bump-according-to-the-pvp/3600

Reply:
I've reported this upstream: https://github.com/erikd/language-javascript/issues/130